docs/siem/reference/timeline-schema.asciidoc

```diff
@@ -0,0 +1,192 @@
+13-digit Epoch timestamp.
+|`updatedBy` |String |The user who last updated the timeline.
+|`version` |String |The timeline's version.
+|==============================================
+
+[[col-obj]]
+[discrete]
+==== columns object
+
+[width="100%",options="header"]
+|==============================================
+|Name |Type |Description
+
+|`aggregatable` |Boolean |Indicates whether the field can be aggregated on all
+indices.
+|`category` |String |The ECS field set to which the field belongs.
+|`description` |String |Column field description.
+|`example` |String |Column field example.
+|`indexes` |String |The list of indices where this field has the same type, or
+null if all indices have the same type for the field.
+|`id` |String |ECS field alias or user defined field name.
+|`name` |String |Reserved for future use, typically `null`.
+|`searchable` |Boolean |Indicates whether this field is indexed for search on
+all indices.
+|`type` |String |???
+|==============================================
+
+[[dataProvider-obj]]
+[discrete]
+==== dataProviders object
+
+[width="100%",options="header"]
+|==============================================
+|Name |Type |Description
+
+|`id` |String |???
+|`name` |String |???
+|`enabled` |Boolean |???
+|`excluded` |Boolean |???
+|`kqlQuery` |String |???
+|`queryMatch` |QueryMatchInput |???
+|`and` |dataProviders |???
+|==============================================
+
+[[eventNotes-obj]]
+[discrete]
+==== eventNotes object
+
+[width="100%",options="header"]
+|==============================================
+|Name |Type |Description
+
+|`created` |Float |The time the note was created, using a 13-digit Epoch
+timestamp.
+|`createdBy` |String |The user who added the note.
+|`eventId` |String |The ID of the event to which the note was added.
+|`note` |String |The note's text.
+|`noteId` |String |The note's ID
+|`timelineId` |String |The ID of the timeline to which the note was added.
+|`updated` |Float |The last time the note was updated, using a
+13-digit Epoch timestamp.
+|`updatedBy` |String |The user who last updated the note.
+|`version` |String |The note's version.
+|==============================================
+
+[[favorite-obj]]
+[discrete]
+==== favorite object
+
+[width="100%",options="header"]
+|==============================================
+|Name |Type |Description
+
+|`favoriteDate` |Float |The time the timeline was marked as a favorite.
+|`fullName` |String |The full name of the user who marked the timeline as
+a favorite.
+|`keySearch` |String |???
+|`userName` |String |The {kib} username of the user who marked the
+timeline as a favorite.
+|==============================================
+
+[[filters-obj]]
+[discrete]
+==== filters object
+
+[width="100%",options="header"]
+|==============================================
+|Name |Type |Description
+
+|`exists` |String |???
+|`meta` |FilterMetaTimelineInput |???
+|`match_all` |String |???
+|`missing` |String |???
+|`query` |String |???
```

Comment:
<img width="492" alt="Screenshot 2020-06-26 at 10 35 39" src="https://user-images.githubusercontent.com/6295984/85843278-df732a00-b798-11ea-8528-9d96ad18f62d.png">
<img width="677" alt="Screenshot 2020-06-26 at 10 35 58" src="https://user-images.githubusercontent.com/6295984/85843271-deda9380-b798-11ea-8268-fd1eef419db9.png">

Reply:
a stringified JSON of current dsl